subvol-mount-generator/suse-btrfs-subvol-mount-generator

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+output="${1:?}"
+
+declare -A mountpoints
+for i in /var /usr/local /srv /root /opt /home /.snapshots; do mountpoints["$i"]=""; done
+
+generate_subvol_mounts() {
+	root_uuid=$(findmnt / -r -n -o UUID)
+	quoted_uuid="${root_uuid//-/\\x2d}"
+	dev="/dev/disk/by-uuid/$root_uuid"
+
+	declare -a line
+	while read -r -a line; do
```

Comment:
This could be replaced with `findmnt -srnoTARGET` and work the other way around by checking that in the loop below

Reply:
Instead of parsing fstab directly? Is there an advantage that compensates for another process fork?

Reply:
Parsing fstab is nontrivial - you can have escape sequences and everything...